### ISSUE TITLE
Support `allPublicMethods` in `org.mikeneck.graalvm.config.ClassUsage`

### ISSUE THREAD

**Issue by sne11ius** [mike-neck/graalvm-native-image-plugin#98]
Describe What happened
===
Task `generateNativeImageConfig` gives this error:
```
Execution failed for task ':mergeNativeImageConfig'.
> org.mikeneck.com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "allPublicMethods" (class org.mikeneck.graalvm.config.ClassUsage), not marked as ignorable (6 known properties: "methods", "allDeclaredFields", "allDeclaredMethods", "fields", "name", "allDeclaredConstructors"])
   at [Source: (BufferedReader); line: 111, column: 26] (through reference chain: org.mikeneck.graalvm.config.ReflectConfig[23]->org.mikeneck.graalvm.config.ClassUsage["allPublicMethods"])
```
===
See reflect-config.json
[reflect-config.json.txt](https://github.com/mike-neck/graalvm-native-image-plugin/files/5648074/reflect-config.json.txt)
===
Used graalvm version:
```
openjdk 11.0.9 2020-10-20
OpenJDK Runtime Environment GraalVM CE 20.3.0 (build 11.0.9+10-jvmci-20.3-b06)
OpenJDK 64-Bit Server VM GraalVM CE 20.3.0 (build 11.0.9+10-jvmci-20.3-b06, mixed mode, sharing)
```

Used command: `./gradlew clean nativeImage`
===
Environment
===

- OS: Ubuntu 20.10
- Plugin Version: v0.9.0
- Gradle Version: 6.7.1

**Comment by mike-neck** [comment]
Thanks for your report.
The fixed feature is available via v0.9.1, which will be released soon.

**Comment by d-led** [comment]
this still seems to be the case in v1.4.1. repo: https://github.com/d-led/hello_vlingo/blob/mikeneck-native-image/build.gradle#L30
tried with graalvm-ce-java11-22.0.0.2, 21.3.0
